css-typed-om/EXPLAINER.md

```diff
@@ -0,0 +1,117 @@
+
+## Spec references
+
+### What is a CSSStyleValue?
+
+[CSSStyleValue](https://drafts.css-houdini.org/css-typed-om-1/#cssstylevalue) objects are the base class through which all CSS values are expressed. For a detailed list of what CSSStyleValue subclasses each CSS property maps to please see [here](PLACEHOLDER). Values that aren't supported as of yet will also be considered as CSSStyleValue objects.
+
+### What are the different CSSStyleValue subclasses?
+
+#### CSSKeywordValue
+
+[CSSKeywordValue](https://drafts.css-houdini.org/css-typed-om-1/#csskeywordvalue) objects represent CSS keywords and other identifiers. An example would be:
+
+```
+element.attributeStyleMap.set("display", new CSSKeywordValue("none")));
+```
+
+#### CSSNumericValue
+
+[CSSNumericValue](https://drafts.css-houdini.org/css-typed-om-1/#cssnumericvalue) objects represent CSS values that are numeric in nature. 
+
+The class can be broken down into two subclasses: 
+* CSSUnitValues represent simple numbers with units - `CSSUnitValue(10, 'px')` for example is the typed representation of `10px`
+* CSSMathValues represent more complex expressions - `CSSMathSum(CSS.em(1), CSS.px(5))` for example is the typed representation of `calc(1em + 5px)`
+
+#### CSSTransformValue
+
+[CSSTransformValue](https://drafts.css-houdini.org/css-typed-om-1/#csstransformvalue) objects represent via [CSSTransformComponents](https://drafts.css-houdini.org/css-typed-om-1/#csstransformcomponent) values, the different functions used by the `transform` property. 
+
+#### CSSPositionValue
+
+[CSSPositionValue](https://drafts.css-houdini.org/css-typed-om-1/#positionvalue-objects) help express the position of a object (e.g. background image) inside a positioning area (e.g. a div).
+
+#### CSSResourceValue
+
+[CSSResourceValue](https://drafts.css-houdini.org/css-typed-om-1/#resourcevalue-objects) objects represent CSS values that require an asynchronous network fetch. Hence, they also describe the status the resource is in. Properties with image values (e.g. `background-image`), are represented by [CSSImageValues](https://drafts.css-houdini.org/css-typed-om-1/#cssimagevalue)
+
+
+### What about Custom properties?
+
+Unregistered custom Properties are represented by [CSSUnparsedValues](https://drafts.css-houdini.org/css-typed-om-1/#cssunparsedvalue) in the Typed OM API. Registered custom properties are represented using [CSSVariableReferenceValues](https://drafts.css-houdini.org/css-typed-om-1/#cssvariablereferencevalue).
+
+## Future Capabilities
+
+The current specification doesn't have support for the following, however they are under consideration for Typed OM Level 2:
+* Support for shorthand properties,
+* Support for gradients,
+* Corresponding FontFaceValue for `@font-face` rules,
+* Support for the following:
+	* Colors
+	* URLs
+	* Shapes, 
+	* Strings, 
+	* Counters, 
+	* etc.
+
+## Conclusion
```

Comment:
Maybe name this "Try it out!" or something *exciting* 🎉

Reply:
Lol Done